controllers/api/BaseApiController.php

```diff
@@ -148,4 +150,18 @@ private function eagerLoadRelations()
             $this->queryBuilder->with($this->eagerLoad);
         }
     }
+
+    private function applySorting()
+    {
+        $sorts = (array) Request::get('sort', null);
```

Comment:
Remove `, null` as it will return `null` by default if the parameter is not found.